packages/dashmate/README.md

```diff
@@ -255,50 +266,18 @@ The `reindex` command works for regular and local configurations.
 
 ```
 Reindex Core
-
 USAGE
   $ dashmate core reindex [-v] [--config <value>]
-
 FLAGS
   -v, --verbose     use verbose mode for output
   --config=<value>  configuration name to use
-
 DESCRIPTION
   Reindex Core
-
   Reindex Core data
 ```
 
-With the hard reset mode enabled, the corresponding config will be reset as well. To proceed, running the node [setup](#setup-node) is required.
 
-To reset a node:
-```bash
-$ dashmate reset
-=======
-#### Hard reset
-
-``dashmate reset --hard``
-
-With the hard reset mode enabled, the corresponding config will be reset as well. This command cleans up all related containers and volumes. To proceed, running the node [setup](#setup-node) is required.
-
-#### Manual reset
-
-Manual reset is used when local setup corrupts and hard reset does not fix it. This could happen, when dashmate configuration becomes incompatible after a major upgrade, making you unable to execute any commands.
 
-```bash
-docker stop $(docker ps -q)
-docker system prune
-docker volume prune
-rm -rf ~/.dashmate/
-```
-
-### Full node
```

Comment:
Did this full node section get accidentally removed? There is still a link to it in the table of contents.

Reply:
It was removed accidentally. Thank you! Fixed that